Cargo.toml

```diff
@@ -18,3 +18,7 @@ derive-try-from-primitive = "1.0"
 mbusparse = { version = "0.1" }
 nom = "7.1"
 serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
Add `default-features = false` here.

Reply:
I added `default-features = false` to all crates needing it